storybook/pages/ActivityFilterMenuPage.qml

```diff
@@ -0,0 +1,174 @@
+import QtQuick 2.14
+import QtQuick.Controls 2.14
+
+import AppLayouts.Wallet.controls 1.0
+import AppLayouts.Wallet.popups 1.0
+
+import StatusQ.Core.Theme 0.1
+import StatusQ.Controls 0.1
+import StatusQ.Popups 0.1
+
+import Storybook 1.0
+
+import Models 1.0
+
+SplitView {
+    id: root
+
+    Logs { id: logs }
+
+    orientation: Qt.Vertical
+
+    QtObject {
+        id: d
+        property int selectedTime: ActivityPeriodFilterSubMenu.All
+        function changeSelectedTime(newTime) {
+            selectedTime = newTime
+            console.warn(" changeSelectedTime = ", newTime)
+        }
+        function setCustomTimeRange(fromTimestamp , toTimestamp) {
+            console.warn(" from timeStamp = ", new Date(fromTimestamp).toISOString())
+            console.warn(" to timeStamp = ", new Date(toTimestamp).toISOString())
```

Comment:
Shouldn't it actually set the range instead of just printing it to console?

Reply:
Done @caybro